shell/platform/darwin/macos/BUILD.gn

```
@@ -248,6 +248,14 @@ copy("copy_framework_module_map") {
   outputs = [ "$_flutter_framework_dir/Versions/A/Modules/module.modulemap" ]
 }
 
+# Copy privacy manifest. This file is required by Apple for third-party SDKs.
+# See https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+copy("copy_framework_privacy_manifest") {
+  visibility = [ ":*" ]
+  sources = [ "framework/PrivacyInfo.xcprivacy" ]
+  outputs = [ "$_flutter_framework_dir/PrivacyInfo.xcprivacy" ]
```

The path is actually incorrect and it was caught during beta codesigning https://github.com/flutter/flutter/issues/157016

The outputs should be changed to 

```suggestion
  outputs =
      [ "$_flutter_framework_dir/Versions/A/Resources/PrivacyInfo.xcprivacy" ]
```

Beta patch was at https://github.com/flutter/engine/pull/55938

---

Let's hold off until we confirm the beta cp passes codesigning.